ruby/README.md

```diff
@@ -0,0 +1,48 @@
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'ci-queue'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install ci-queue
+
+## Usage
+
+### Supported CI providers
+
+`ci-queue` automatically infer most of it's configuration if ran on one of the following CI providers:
+
+  - Buildkite
+  - CircleCI
+  - Travis
+
+If you are using another CI system, please refer to the command usage message.
+
+### Minitest
+
+Assuming you use one of the supported CI providers, the command can be as simple as:
+
+```bash
+minitest-queue --url redis://example.com run -Itest test/**/*_test.rb
```

Comment:
I find the meaning of `--url` a bit vague. Maybe instead of using `--url` we should use `--queue` ? Not sure.

Reply:
Or, `--orchestrator` or `--coordinator`

Reply:
Changed it for `--queue`